jaeger-core/src/main/java/com/uber/jaeger/Span.java

```diff
@@ -223,6 +225,15 @@ public Span log(long timestampMicroseconds, Map<String, ?> fields) {
         if (logs == null) {
           this.logs = new ArrayList<LogData>();
         }
+
+        if (tracer.expandExceptionLogs()) {
+          Map<String, Object> errorFields = exceptionLogs(fields);
+          if (!errorFields.isEmpty()) {
+            errorFields.putAll(fields);
+            fields = errorFields;
```

Comment:
I'm not overly familiar with certain Java things, but given that we do `errorFields.putAll(fields)` above, is it still necessary for `fields = errorFields;`?